utilities/backupable/backupable_db.cc

```diff
@@ -670,6 +671,9 @@ BackupEngineImpl::~BackupEngineImpl() {
     t.join();
   }
   LogFlush(options_.info_log);
+  for (const auto& it : corrupt_backups_) {
+    it.second.first.PermitUncheckedError();
```

Comment:
I have the same question about this being in the destructor as before

Reply:
There does not appear to be code that makes sure there are no corrupt backups prior to shutting down, hence the need for a check here.  I will add a comment